src/main/java/ch/njol/skript/expressions/ExprCoordinate.java

```diff
@@ -32,19 +28,22 @@
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.util.Kleenean;
+import org.bukkit.Chunk;
+import org.bukkit.Location;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Coordinate")
-@Description("Represents a given coordinate of a location. ")
+@Description("Represents a given coordinate of a location/chunk. ")
 @Examples({"player's y-coordinate is smaller than 40:",
-		"	message \"Watch out for lava!\""})
-@Since("1.4.3")
-public class ExprCoordinate extends SimplePropertyExpression<Location, Number> {
+		"	message \"Watch out for lava!\"",
+		"player's chunk's z-coordinate is bigger than 10",
+		"	message \"You're leaving the underground!\""})
+@Since("INSERT VERSION")
```

Comment:
As per Ayham's previous comment, this still needs to be updated!

```suggestion
@Since("1.4.3, INSERT VERSION (chunks)")
```